frida_tools/tracer.py

```diff
@@ -52,7 +52,8 @@ def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.exclude_objc_method,))
             parser.add_option("-s", "--include-debug-symbol", help="include DEBUG_SYMBOL", metavar="DEBUG_SYMBOL",
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_debug_symbol,))
-            parser.add_option("-q", "--quiet", help="do not format agent's output", action='store_true')
+            parser.add_option("-q", "--quiet", help="do not format output messages", action='store_true', default=False)
+            parser.add_option("-o", "--output", help="dump messages to file", metavar="OUTPUT", type='string', default="")
```

Comment:
```suggestion
            parser.add_option("-o", "--output", help="dump messages to file", metavar="OUTPUT", type='string')
```

Reply:
so it's just the string type that have a meaningful defaults? sholdnt we set also 'boolean' type in the -q and drop the default=False?